webpack.config.js

```diff
@@ -44,12 +44,15 @@ module.exports = {
         test: /\.scss$/,
         loader: 'style!css!sass'
       },
-      {test: /\.png$/, loader: "url-loader?mimetype=image/png"},
+      {test: /\.(png|jpg)$/, loader: 'url-loader?limit=8192'},
       {test: /\.css$/, loader: ExtractTextPlugin.extract('style-loader', 'css-loader!cssnext-loader')},
       {test: /\.js$/, loaders: ['react-hot', 'babel'], exclude: /node_modules/}
     ]
   },
   cssnext: {
-    browsers: 'last 2 versions'
+    browsers: 'last 2 versions',
+    plugins: [
+      require('postcss-nested')
```

Comment:
And needs to be correct version: https://github.com/codefordenver/fresh-food-connect/commit/83df831d0394b575371fa6045a6180d401213811